src/prometheus_alert_rules/reload_failure.rule

```diff
@@ -0,0 +1,8 @@
+alert: TraefikConfigReloadFailure
+expr: rate(traefik_config_reloads_failure_total[3m]) > bool 0
+for: 1m
```

Comment:
Might be handy to have a docstring for the rationale.
For example, why not `increase(traefik_config_reloads_failure_total[3m]) > 0` `for 1m`?

Reply:
I like it, seems more explicit than my initial version. Thanks!